src/marten/cli/manage/command/collect_assets.cr

```diff
@@ -6,10 +8,21 @@ module Marten
           command_name :collectassets
           help "Collect all the assets and copy them in a unique storage."
 
+          @fingerprint : Bool = false
+          @fingerprint_mapping = Hash(String, String).new
           @no_input : Bool = false
+          @manifest_path : String = File.join Marten.apps.main.class._marten_app_location, "manifest.json"
```

Comment:
Is it intended that this file will be created at the root of the project by default (and not in the `src` folder?)?

Reply:
This resolves to `src/manifest.json` for me 🤔 but if we are already talking about it, is there a better variable for it? Because this setting resolves to the `marten/src` instead of the test project directory when running the specs without changing this value.